### ISSUE TITLE
Notch is not centered

### ISSUE THREAD

**Issue by thatrawk** [hereafter/dot-notch#1]
The notch that this project adds is not centered, and is slightly to the right of the screen.

**Comment by hereafter** [comment]
please kindly post a picture :)

**Comment by thatrawk** [comment]
Notch not centered, as shown in these pictures. Both webcams are in the center of the screen, and the notch is slightly to the right. If you can figure out how to fix it, please roll out an update through Microsoft Store, liking the notch so far except for this issue

![IMG_3211](https://user-images.githubusercontent.com/103553152/205581269-7afc9865-55f5-4328-b7ab-87dbfe1efc09.JPG)
![IMG_3212](https://user-images.githubusercontent.com/103553152/205581303-fdd80fc8-b2f9-4cec-9c18-907bbcfa1a22.JPG)

**Comment by hereafter** [comment]
i see, i ll look into the issue

**Comment by derek00001** [comment]
Still not center, mind to fix ?

**Comment by swanlon** [comment]
Yeah, still not fixed, when changing DPI scale you can see the notch trying to move center but it's quickly offset to the right
